utilsForFrontend.mjs

```diff
@@ -475,7 +473,14 @@ export function isVisible(fnodeOrElement) {
     if (elementStyle.visibility === 'hidden') {
         return false;
     }
+    // Check if the element is off-screen
```

Comment:
I was saying to Daniel the other day that I like to punctuate comments like sentences if they are, in fact, sentences. I think it's influence from https://www.python.org/dev/peps/pep-0008/#comments. But on comments like this that take up the whole line and refer to code below, I often stick a colon afterward. That way, even if you don't skip a line before, it's clear you're talking about code only below it.

Reply:
I can get behind the complete sentence thing. Not sure if I will remember your preference around colons, but I'll try to keep that in mind...I added a colon.